webrender/res/prim_shared.glsl

```diff
@@ -727,11 +727,12 @@ TransformVertexInfo write_transform_vertex(RectWithSize instance_rect,
 
 struct ResourceRect {
     vec4 uv_rect;
+    vec2 user_data;
```

Comment:
Does it have to be `ResourceRect` then? Maybe we can have `GlyphRect` struct for this.